app/controllers/api/v1/targets_controller.rb

```diff
@@ -4,6 +4,11 @@ class TargetsController < ApplicationController
 
       before_action :authenticate_user!
 
+      def index
+        @targets = Target.where(user_id: current_user.id)
```

Comment:
```suggestion
        @targets = current_user.targets
```
Take a look at [this](https://github.com/heartcombo/devise#controller-filters-and-helpers).